### ISSUE TITLE
ambiguity of the available attribute

### ISSUE THREAD

**Issue by walters301** [TAXIIProject/TAXII-Specifications#72]
Under the Collection element of the Collection_Information_Response, there is an optional attribute called "@available".  Assuming you have an implementation of TAXII that is built upon feeds/collections and subscriptions, if there is a Collection_Information_Response with the @available attribute set to true, should that mean that the user is subscribed? and false would mean they aren't subscribed?  Or should true mean that it's available for subscribing and false means they don't have access to it at all?  My thought is that under the former case, when users see available=false they may think they don't have the ability to subscribe to it and therefore may not attempt.  Similarly, in the latter case, when a user sees available=true, they may believe they're already subscribed and will encounter errors when they try to poll.

**Comment by jordan2175** [comment]
Good catch.  Yes, that will be an issue.  @MarkDavidson can we add a label to this issue to address and fix in either TAXII 1.2 or TAXII 2.0.  Not sure if the clarification will result in a breaking change or not.

**Comment by MarkDavidson** [comment]
@jordan2175,

What label were you thinking of? I think there's an implicit assumption that all issues in this tracker are considerations for any future revision of TAXII.

-Mark
